### PR TITLE
Make export to UVFits work with non-constant receptor angles

### LINKED ISSUE #1340: MSFits: non-constant receptor angles in MS FEED table prevent export to UV FITS
In an ALMA helpdesk ticket, a user reported a case where an MS contained non-constant receptor angles in the FEED table and could therefore not be exported.
These angles are only relevant for calibration. If the date is already calibrated, which is usually the case before export to UV FITS, the angles are irrelevant.
I propose to relax the conditions of the code and make it only produce a warning when non-constant (in time) receptor angles are encountered.

### PR DESCRIPTION
**Pull request by tammojan**
Implemented by Dirk Petry (@petry)

Fixes #1340

### CODE REVIEW

**Comment by dpetry** [comment]
@tammojan , looks like the MacOS failure is not related to the actual code change ...

**Comment by tammojan** [comment]
> @tammojan , looks like the MacOS failure is not related to the actual code change ...

Agree. Not the first time we have to babysit the MacOS build. In the meantime, this build failure should not delay fixing your issue. Do you have a volunteer to review this change?

**Comment by dpetry** [comment]
@kettenis , hi, would you mind reviewing this little adjustment? It downgrades the reaction to a potential problem
in the MS-to-UVFITS (not FITS-IDI) translation of the FEED table from an exception to a Warning.
The warning text explains it.

**Comment by dpetry** [comment]
@tammojan , not sure if Mark (Kettenis) is actually entitled to do a review. Is he? If not, could you pick a different reviewer?

File: msfits/MSFits/MSFitsOutput.cc [hunk 1]
```diff
@@ -1871,10 +1871,19 @@ Bool MSFitsOutput::_writeAN(std::shared_ptr<FitsOutput> output, const Measuremen
                         antToRA[antnum] = ra;
                     }
                     else {
-                        _checkReceptorAngles(ra, antToRA[antnum], antnum);
+                        try {
+                            _checkReceptorAngles(ra, antToRA[antnum], antnum);
+                        }
+                        catch (const std::exception& x) {
+                            os << LogOrigin("MSFitsOutput", __func__)
+                               << LogIO::WARN << "Receptor Angles in FEED table are not constant for antenna " << antnum
```
Comment (apschoenmakers) [inline msfits/MSFits/MSFitsOutput.cc:1879]: constant in time?

**Review by apschoenmakers — APPROVED** [review]

**Pull request merged**